com.unity.render-pipelines.universal/Runtime/Data/PostProcessData.cs

```diff
@@ -69,6 +69,9 @@ public sealed class ShaderResources
             [Reload("Shaders/PostProcessing/LensFlareDataDriven.shader")]
             public Shader LensFlareDataDrivenPS;
 
+            [Reload("Shaders/PostProcessing/UpscaleSetup.shader")]
+            public Shader upscaleSetupPs;
```

Comment:
Minor suggestion for consistency with other naming.

```suggestion
            public Shader upscaleSetupPS;
```